contributing.md

```diff
@@ -0,0 +1,59 @@
+# Contributing to {{Project-Name}}
+
+We welcome everyone to contribute what they can. Whether you are brand new, just want to contribute a little bit, or want to contribute a lot there is probably something you can help out with.
+
+## Where to get help
+The main way to get help is on our [discord server](https://discord.gg/uh69TdKfBD).  Though, this guide should help you get started. It may be slightly lengthy, but it's designed for those who are new so please don't let length intimidate you.
```

Comment:
double-checking, is the link here an infinite invite to the general lobby/channel?

Reply:
Yes, it is